docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Announcing Gatsby v2 beta launch!
+date: "2018-06-14"
+author: "Mike Allanson"
+tags: ["v2", "gatsby"]
+---
+
+We’re excited to announce that today we shipped the first beta for Gatsby v2! 35 contributors have made over 300 commits to v2 since Gatsby v1 was released in July 2017.
+
+## What's coming in v2?
+
+Gatsby v2 builds on the foundations of v1 to introduce a range of improvements:
+
+- Out With Layouts, In With StaticQuery
```

Comment:
Let's say "Gatsby Layout components"  (or something equivalent) — "layouts" still exist in v2 — they're just your components now not a special things.